Gruntfile.js

```diff
@@ -412,8 +396,7 @@ module.exports = function (grunt) {
   ]);
 
   grunt.registerTask('server', [
-    'jsbeautifier:html',
-    'csscomb',
+    'htmllint',
```

Comment:
Here linting is done in the `watch` task so don't really need to do it before calling `watch`. Also if the linting fails here the `watch` task won't run which means running the task manually everytime until it's fixed.